packages/squiggle-lang/src/public/SqValue/SqDict.ts

```diff
@@ -19,7 +19,7 @@ export class SqDict {
   }
 
   get(key: string): SqValue | undefined {
-    const value = this._value.get(key);
+    const value = this._value.get(vString(key));
```

Comment:
So this goes through `VDict.get()`, I guess that's more correct than before, even if it does the same thing for now.

Reply:
yea. It's a a bit weird, but seemed more consistent.